subiquity/models/filesystem.py

```diff
@@ -665,13 +663,13 @@ def annotations(self):
         return []
 
     def desc(self):
+        if self.multipath:
+            return "multipath"
```

Comment:
I wonder if _("multipathed disk") would be better here.

Reply:
I've seen 'multipath member device' used (Centos/Fedora).

Reply:
The thing being labelled is the whole device though, not a member (conceptually speaking, anyway)

Reply:
Multipath manpage uses "multipath device" and "multipath device map", as in using multipath as as adjective to describe a device.

Windows world uses MPIO moniker for MultiPath I/O.

Freebsd uses "multipath device" too in its gmultipath util.

I do prefer device, because what's on the other end might not be "a disk".

Solaris seems to call it MPxIO.

I think going with "multipath device" is short enough, and universal enough for our purposes.